src/app/content/components/ContentLink.tsx

```diff
@@ -31,6 +33,7 @@ interface Props {
   currentPath: string;
   hasUnsavedHighlight: boolean;
   search: RouteState<typeof content>['search'];
+  scrollTarget?: Exclude<ReturnType<typeof selectNavigation.scrollTarget>, null>;
```

Comment:
If we allow passing `null` `scrollTarget` it would seem like we do something with it, but in reality, we would omit it as we would do with `undefined`